### PR TITLE
Updated README.md with info about simulation. 

### PR DESCRIPTION
**Pull request by KRaizer**
Also, added a new README.md inside simulated-environment for extra instructions.

### CODE REVIEW

File: simulated-enviroment/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Access to this scene (both reading and control) is currently implemented by using VREP's remoteAPI in python. This is expected to change once [ROS](http://www.ros.org/) is deployed and used for controlling the robots in the scene.
+
+## Deployment
+
+
+Step 1:  Download and install v-rep simulator#
+
+Go to : http://www.coppeliarobotics.com/downloads.html
+
+and download the V-REP Pro Edu version appropriate for your operational system, e.g. Linux 64 bits.
```
Comment (berezovskyi) [inline simulated-enviroment/README.md:16]: is player enough to run?

**Review by berezovskyi — APPROVED** [review]

**Comment by berezovskyi** [comment]
image

File: simulated-enviroment/README.md (hunk 1 above)
Comment (KRaizer) [reply simulated-enviroment/README.md:16]: Yes! I'll add that on the next commit.

Comment (berezovskyi) [reply simulated-enviroment/README.md:16]: You can add this commit to the same branch, push it, and the pull request here will get automatically updated.

**Pull request merged**